pfio/v2/fs.py

```diff
@@ -462,7 +431,7 @@ def _from_scheme(scheme, dirname, kwargs, bucket=None):
         from .s3 import S3
         fs = S3(bucket=bucket, prefix=dirname, **kwargs)
     else:
-        raise RuntimeError("bug: scheme '{}' is not supported".format(scheme))
+        raise RuntimeError("scheme '{}' is not supported".format(scheme))
```

Comment:
Maybe reaching this line has not been a bug, but then "not supported" is too strong regarding the spec schemes can be dynamically added. Maybe change it as "not defined"?

Reply:
Fixed in 13319b2638a4a5124e9cdab5001324cb2a90134f.